mesonbuild/compilers/swift.py

```diff
@@ -115,6 +117,29 @@ def get_include_args(self, path: str, is_system: bool) -> T.List[str]:
     def get_compile_only_args(self) -> T.List[str]:
         return ['-c']
 
+    def get_options(self) -> MutableKeyedOptionDictType:
+        opts = super().get_options()
+
+        key = self.form_compileropt_key('lang_version')
```

Comment:
I'd prefer if we use `swift_std` here. Rust technically calls their thing "editions", but in order to get a consistent interface we went with `std`